org.eclipse.jdt.ls.tests/src/org/eclipse/jdt/ls/core/internal/managers/ProjectsManagerTest.java

```diff
@@ -36,6 +48,25 @@
 
 	private JavaLanguageClient client = mock(JavaLanguageClient.class);
 	private JavaClientConnection javaClient = new JavaClientConnection(client);
+	private JDTLanguageServer server;
+
+	@Before
+	public void setup() throws Exception {
+		server = new JDTLanguageServer(projectsManager, preferenceManager);
```

Comment:
you need to store the workspace autobuilding value

Reply:
done